src/db/models/user.js

```diff
@@ -12,10 +11,6 @@ module.exports = (sequelize, DataTypes) => {
     clientId: DataTypes.STRING
   }, {});
   User.associate = (models) => {
-    User.hasMany(models.Request, {
```

Comment:
Why has this association been removed? It may be needed.

Reply:
I have added them in the new migrations I have created to update both the requests and users tables.